### ISSUE TITLE
gl_MaxVaryingVectors Built-In Constants meets a compiler error.

### ISSUE THREAD

**Issue by ZhiqianXia** [KhronosGroup/glslang#2682]
Hi, the  **gl_MaxVaryingVectors** builtInConstant will get a compiler error for non-es profile. I think it should be add 
at TBuiltIns::initialize.

**shader:**
```
#version 450
layout(local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
layout(std430) buffer Output {
  int data; 
} g_out;
void main() 
{ 
  g_out.data = gl_MaxVaryingVectors; 
}
```

**Comment by greg-lunarg** [comment]
Agreed. Will you be submitting a PR?

**Comment by greg-lunarg** [comment]
Fixed by #2684
